### PR TITLE
Added examples - and changes to the API endpoints to match with the ones on the online doc

### PR DESCRIPTION
**Pull request by wardhanster**

### CODE REVIEW

File: src/Mobius.php [hunk 1]
```diff
@@ -53,10 +53,16 @@ public function registerToken($type = 'ERC20', $name, $symbol, $address)
...
+        // if (!isset($response['token_uid'])) throw new \ErrorException('Unexpected API response: ' . print_r($response, true));
 
-        if (!isset($response['token_uid'])) throw new \ErrorException('Unexpected API response: ' . print_r($response, true));
-
-        return $this->getToken($response['token_uid']);
+        // return $this->getToken($response['token_uid']);
+        
+        /* ADDED BY HARSH 
+                - the api does not seems to be returning token_uid - it returns uid: hence using that instead 
+                @david needs to tell us if this is something we need to merge in the original codebase
```
Comment (dgobaud) [inline src/Mobius.php]: @wardhanster @zulucrypto confirmed this is now just uid in our api so this should be ready to merge!

Comment (dgobaud) [reply src/Mobius.php]: @wardhanster maybe remove the commend now that we have confirmation - the docs on our site are being updated

**Comment by zulucrypto** [comment]
@wardhanster Thanks for updating the API and adding the examples!

Before merging, could you remove the `.DS_Store` files as well as the comment now that @dgobaud has confirmed the parameter?

**Comment by wardhanster** [comment]
@zulucrypto Yes sure 
I will do that shortly !

**Comment by zulucrypto** [comment]
@dgobaud - I've cleaned up the `.DS_Store` files and the comment, is this OK to merge now?

**Comment by dgobaud** [comment]
looks good! thanks

**Pull request merged**